microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -18,12 +18,21 @@ In addition to loading remote content, content can also be loaded locally into W
 
 These approaches are described below.
 
+| Consideration | Navigating to a file URL | Navigating to an HTML string | Virtual host name mapping | `WebResourceRequested` |
+| --- | --- | --- | --- | --- |
+| Origin based DOM APIs | ✔️ | ❌ | ✔️ | ✔️ |
+| DOM APIs requiring secure context | ❌ | ❌ | ✔️ | ✔️ |
+| Dynamic content | ❌ | ✔️ | ❌ | ✔️ |
+| Additional web resources | ✔️ | ❌ | ✔️  | ✔️ |
+| Additional web resources resolved in WebView2 process | ✔️ | ❌ | ✔️ | ❌ |
+
 
 <!-- ====================================================================== -->
 ## Navigate to a file URL
 
 WebView2 allows navigations to file URLs, to load basic HTML or a PDF.  This is the simplest and most efficient approach to loading local content.  However, it is less flexible than the other approaches.  Like in a web browser, file URLs are limited in some capabilities:
-*  The document origin will be `null` for a file URL.
+*  The document has an origin that is unique to each file, just like in the browser. So APIs that require an origin such as `localStorage`, `indexedDB`, and others will work, but different file URL documents are not considered same origin.
```

Comment:
```suggestion
*  The document has an origin that is unique to each file, just like in the browser. So APIs that require an origin such as `localStorage` or `indexedDB` will work, but different file URL documents are not considered "same origin".
```
* "such as" is redun. w/ "and others".  simplified.
* feel the special tech term warrants offset-quotes here.